specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```
@@ -0,0 +1,2962 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Load Testing",
+    "description": "These APIs allow end users to create, view and run load tests using Azure Load Test Service.",
+    "version": "2022-11-01"
+  },
+  "paths": {
+    "/appcomponents/{name}": {
```

Should use kebab-case in paths.
```suggestion
    "/app-components/{name}": {
```

---

changed